login-fire-button.html

```diff
@@ -130,15 +131,24 @@
         @apply --login-fire-btn-signout-hover;
       }
 
+      .btn-progress {
```

Comment:
This CSS class can be moved into the `login-fire-styles` file and the shared styles included in the `login-fire-form` element. But is it really necessary to import the shared styles in the form just to have the progress bar one?... 🤔

Reply:
I will move it to the shared styles files, as it is better to maintain the default styling at a central place instead of copying it.